packages/cms-data-sync/src/utils/rich-text.ts

```diff
@@ -111,6 +119,7 @@ export const convertHtmlToContentfulFormat = (html: string) => {
   processedHtml = removeStylingTagsWrappingIFrameTags(processedHtml);
   processedHtml = removeStylingTagsWrappingImgTags(processedHtml);
   processedHtml = wrapIframeWithPTag(processedHtml);
+  processedHtml = wrapPlainTextWithPTag(processedHtml);
 
   // processedHtml = addPTagsToIframeTagNotWrappedByAnything(processedHtml);
```

Comment:
I think I've left this comment by mistake 🙈 could you erase before merging?

Reply:
Do you mean uncomment, or remove the line completely?

Reply:
remove it completely :)